src/main/java/pageObjects/WebPages/GooglePage.java

```diff
@@ -0,0 +1,64 @@
+package pageObjects.WebPages;
+
+import io.appium.java_client.AppiumDriver;
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class GooglePage extends WebPageObject {
+
+    public static final String googleURL = "https://www.google.com";
+
+    public WebElement getSearchField() {
+        return searchField;
+    }
+
+    @FindBy(xpath = "//input[@name='q']")
+    private WebElement searchField;
+
+    @FindBy(xpath = "//*[@id='rso']/*")
+    private List<WebElement> searchResults;
+
+    //Change "Olvasson" to your scroll down button text if it differs
+    public static final String scrollXpath = "//button//*[contains(text(),'Olvasson')]";
+
+    //Change "eluta" to your accept cookie button text if it differs
+    public static final String acceptButtonXpath = "//button//*[contains(text(),'eluta')]";
+
+    @FindBy(xpath = scrollXpath)
+    private WebElement scrollButton;
+
+    @FindBy(xpath = acceptButtonXpath)
```

Comment:
It looks like it would be better to use the exact xpath string in FindBy, not a variable

Reply:
Fixed